cheatsheets/Microservices_based_Security_Arch_Doc_Cheat_Sheet.md

```diff
@@ -0,0 +1,148 @@
+# Introduction
+
+The microservice architecture is being increasingly used for designing and implementing application systems in both cloud-based and enterprise infrastructures. There are many security challenges need to be addressed in the application design and implementation phases. In order to address some security challenges it is necessity to collect security-specific information on application architecture.
+The goal of this article is to provide a concrete proposal of approach to collect microservice-based architecture information to securing application.
+
+# Context
+
+During securing applications based on microservices architecture, security architects/engineers usually face with the following questions (mostly referenced in the OWASP Application Security Verification Standard Project under the section V1 “Architecture, Design and Threat Modeling Requirements”):

```

Comment:
Add links to ASVS e.g.

mostly referenced in the [OWASP Application Security Verification Standard Project](https://github.com/OWASP/ASVS) under the section
[V1 “Architecture, Design and Threat Modeling Requirements”](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#v1-architecture-design-and-threat-modeling-requirements)

Reply:
Fixed